src/components/Overview/OverviewGrid.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PercentageOverview from './PercentageOverview';
+import { useRouter } from 'next/router';
+import { Wrapper } from '../UI/PercentageOverview/Wrapper.styled';
+import useStore from '../../lib/useStore';
+import { NavButton } from '../UI/Button/NavButton.styled';
+import SvgIcon from '../SVGs/icons';
+import ChartIllustration from '../SVGs/illustrations/illustrations';
+export default function OverviewGrid() {
+	const didIts = useStore(state => state.didIts);
+	const todos = useStore(state => state.todos);
+	const doingIts = useStore(state => state.doingIts);
+
+	const lengthOfTodos = todos.length;
+	const legthOfDoingIts = doingIts.length;
+	const lengthOfDidIts = didIts.length;
+	const sumOfArrays = lengthOfTodos + legthOfDoingIts + lengthOfDidIts;
+
+	const progressTodo = Math.round(
+		(lengthOfTodos / (lengthOfTodos + legthOfDoingIts + lengthOfDidIts)) * 100
```

Comment:
Why don't you use `sumOfArrays` (or `sumOfTasks` if you decide to change it)?

```suggestion
		(lengthOfTodos / sumOfArrays) * 100
```